lib/components/progresso.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+const _mensagem = 'Carregando...';
+
+class Progresso extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
```

Comment:
`Center` is mostly a layout widget, and probably should be used at the main widget that is building it.
Unless you are explicitly telling this widget to always be in the center of any container it is in?

Reply:
I used this Widget because, in the `ListaTransferencias`, I fill the screen with `Progresso` Widget while try load new transfers. Considering this necessity, all right keep this approach? And I'm reusing these Widget while I try load new features too

Reply:
It was just to make sure this `Center` widget belongs here or not 👍